packages/flutter/test/foundation/caching_iterable_test.dart

```diff
@@ -107,4 +107,19 @@ void main() {
     expect(expanded2, equals(<int>[1, 1, 2, 2, 3, 3, 4, 4, 5, 5]));
     expect(yieldCount, equals(5));
   });
+
+  test('The Caching Iterable: elementAt correctness', () {
```

Comment:
I think "The Caching Iterable" looks kinda silly, but since it's following a pattern from other tests in this file it's all good 👍